0-wrappers/FCCvote-net-neutrality/style/style.css

```diff
@@ -0,0 +1,597 @@
+html,
+body {
+  font-size: 62.5%;
+}
+body {
+  font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: 300;
+  font-size: 1.1rem;
```

Comment:
@gesa this was based on Sabrina's mockup.  Would you recommend a bigger font size like 12px or something?  Not sure what the convention is.